tests/OBO.NET.CodeGeneration.Tests/CodeGeneration.Tests.fs

```diff
@@ -0,0 +1,55 @@
+namespace OBO.NET.CodeGeneration.Tests
+
+open Expecto
+open OBO.NET
+open OBO.NET.CodeGeneration
+open FSharpAux
+open ARCTokenization.StructuralOntology

```

Comment:
i would not use these ontologies but static files in this repo as references for tests